api/src/mailbox.rs

```diff
@@ -1308,6 +1322,113 @@ impl Request for ProductionAuthDebugUnlockToken {
     type Resp = MailboxRespHeader; // TODO Check
 }
 
+pub const CMK_MAX_KEY_SIZE_BITS: usize = 512;
+pub const CMK_SIZE_BYTES: usize = 128;
+/// CMK is an opaque (encrypted) wrapper around a key.
+#[derive(Clone, Debug, FromBytes, Immutable, IntoBytes, KnownLayout, PartialEq, Eq)]
+pub struct Cmk(pub [u8; CMK_SIZE_BYTES]);
+
+impl Default for Cmk {
+    fn default() -> Self {
+        Self([0u8; CMK_SIZE_BYTES])
+    }
+}
+
+#[repr(u32)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum CmKeyUsage {
```

Comment:
Just for my understanding, is import only intended for symmetric keys? And ECDH/ML-KEM keys should be generated inside Caliptra?

Reply:
I think for now, we should only import symmetric keys (AES, HMAC / HKDF secrets). Probaly MLDSA seeds would be fine too, once IETF standardizes the format (this is currently a hotly debated topic, from what I gather). Definitely we want to avoid importing raw ECDSA keys, since they are full of weird things we have to check for.

Reply:
lgtm. I think for how they get used in most protocols, generating asymmetric keys internally only makes sense.